routes/student.js

```diff
@@ -1,24 +1,27 @@
 const { response } = require("express");
 const db = require("../database/db");
+const program_db = require("../database/program_db");
 
 module.exports = {
-	addStudentPage: function (request, response) {
+	addStudentPage: async function (request, response) {
+		let programList = await program_db.getProgramsList();
 		let renderData = {
 			student: {},
 			add: true,
-			view: false
+			view: false,
+			programs: module.exports.activePrograms(programList),
 		};
 
 		response.render('edit-student', renderData);
 	},
 	viewStudentPage: async function (request, response) {
 		let studentId = request.params.id;
 		let studentObj = await db.getStudentById(studentId);
-		
 		let renderData = {
 			student: studentObj,
 			add: false,
-			view: true
+			view: true,
+			programs: studentObj.program_list,
```

Comment:
Similarly to above, after updating `program_list` to use ObjectIDs, you'll have to get the program objects from their IDs